useScroll/useScroll.js

```diff
@@ -485,7 +485,7 @@ const useScroll = (props) => {
 	});
 
 	assignProperties('hoverToScrollProps', {
-		scrollContainerHandle,
+		scrollContainer: scrollContainerHandle.current,
```

Comment:
Let's rename this property to `scrollContainerHandle` or `scrollContainerHandleRef` to be clearer.